README.md

```diff
@@ -710,9 +709,8 @@ Other Style Guides
     }
 
     // good
-    let test;
     if (currentUser) {
-      test = () => {
+      let test = () => {
```

Comment:
this change isn't wrong, but is immaterial to the example - why is it an improvement?